<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cadastro</class>
 <widget class="QMainWindow" name="Cadastro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>369</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>347</width>
    <height>369</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>347</width>
    <height>369</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cadastre-se</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(57, 130, 195);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lbl_crie_conta">
      <property name="styleSheet">
       <string notr="true">font-size: 20pt;
color: white;
font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Crie sua conta</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QLineEdit" name="nome">
      <property name="minimumSize">
       <size>
        <width>170</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>170</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">*{
background-color: #fff;
border-radius: 15px;
padding: 5px;
padding-left: 7px;
}
QLineEdit:focus{
border: 2px solid #edd400;
}</string>
      </property>
      <property name="maxLength">
       <number>20</number>
      </property>
      <property name="placeholderText">
       <string>Nome</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QLineEdit" name="usuario">
      <property name="minimumSize">
       <size>
        <width>170</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>170</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">*{
background-color: #fff;
border-radius: 15px;
padding: 5px;
padding-left: 7px;
}
QLineEdit:focus{
border: 2px solid #edd400;
}</string>
      </property>
      <property name="maxLength">
       <number>20</number>
      </property>
      <property name="placeholderText">
       <string>Usu√°rio</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QLineEdit" name="senha">
      <property name="minimumSize">
       <size>
        <width>170</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>170</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">*{
background-color: #fff;
border-radius: 15px;
padding: 5px;
padding-left: 7px;
}
QLineEdit:focus{
border: 2px solid #edd400;
}</string>
      </property>
      <property name="maxLength">
       <number>20</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Senha</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbl_erro">
      <property name="styleSheet">
       <string notr="true">color: rgb(237, 212, 0);;
font-size: 13pt;
font-weight: bold;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="btn_cadastrar">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">*{
background-color: #edd400;
border-radius: 15px;
}
QPushButton:hover{
background-color: #d1bc08;
}

</string>
      </property>
      <property name="text">
       <string>Cadastrar</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="btn_voltar">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">*{
background-color: #6EC8DB;
border-radius: 15px;
color: white;
}
QPushButton:hover{
background-color:#60afbf;
}</string>
      </property>
      <property name="text">
       <string>Voltar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>usuario</tabstop>
  <tabstop>senha</tabstop>
  <tabstop>btn_cadastrar</tabstop>
  <tabstop>btn_voltar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
